src/ctapipe/utils/astro.py

```diff
@@ -18,8 +18,18 @@
 
 
 class StarCatalogues(Enum):
-    Yale = "V/50/catalog"  #: Yale bright star catalogue
-    Hipparcos = "I/239/hip_main"  #: hipparcos catalogue
+    Yale = {
+        "directory": "V/50/catalog",
+        "band": "V",
+        "frame": "J2000",
+        "ID_type": "HR",
+    }  #: Yale bright star catalogue
+    Hipparcos = {
+        "directory": "I/239/hip_main",
+        "band": "V",
```

Comment:
Hipparcos has both V and B band, why specify V here?

Reply:
i made it now so that both B and V can be used in one catalog